interoperator/pkg/webhook/default_server/server.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2018 The Service Fabrik Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package defaultserver
+
+import (
+	"fmt"
+	"os"
+
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission/builder"
+)
+
+var (
+	log        = logf.Log.WithName("default_server")
+	builderMap = map[string]*builder.WebhookBuilder{}
+	// HandlerMap contains all admission webhook handlers.
+	HandlerMap = map[string][]admission.Handler{}
+)
+
+// Add adds itself to the manager
+func Add(mgr manager.Manager) error {
+	// TODO: find a better way to initialize the values below; maybe read from
```

Comment:
Does this comment make sense ?

Reply:
Currently the server `namespace`, `port` and `CertDir` are being hard-coded. I'll be discussing alternatives IRL, the comment is just a reminder in case I forget.